website/contributing/release-stable-patch.md

```diff
@@ -27,13 +27,42 @@ git pull origin <release-branch> --tags
 git cherry-pick <commit>
 ```
 
-### 2. Test the current changes
+### 2. Bump monorepo packages
```

Comment:
@kelset can you double check the new steps are correct both for 0.72 and 0.71? 🙏